### ISSUE TITLE
ContractConstructor exceptions fix

### ISSUE THREAD

**Issue by Badiboy** [andelf/tronpy#15]
The field "name" definitely does not exist in class ContractConstructor.

![image](https://user-images.githubusercontent.com/5613421/105626909-472f4780-5e44-11eb-9e69-8e7d553239b6.png)

https://github.com/andelf/tronpy/blob/531cd317801ee4391214b623277d014efc17ba20/tronpy/contract.py#L216

**Comment by Badiboy** [comment]
Ups, PyCharm fault. :)

```
    @property
    def name(self) -> str:
        return self._abi["name"]

```

**Comment by Badiboy** [comment]
Should add

```
    @property
    def name(self) -> str:
        return self._abi["name"]

```
to ContractConstructor?

**Comment by andelf** [comment]
Thanks. I will fix this soon.

**Comment by andelf** [comment]
Released as v0.2.4
